.circleci/config.yml

```diff
@@ -484,6 +484,23 @@ jobs:
       # Run project tests with NGC and View Engine.
       - run: bazel test src/... --build_tag_filters=-docs-package,-e2e --test_tag_filters=-e2e --config=view-engine --build_tests_only
 
+  # ----------------------------------------------------------------------------
+  # Job that runs all Bazel integration tests.
+  # ----------------------------------------------------------------------------
+  integration_tests:
+    <<: *job_defaults
+    resource_class: xlarge
```

Comment:
I think we should do some audits on resource classes in the future. Currently we have them set in all expensive jobs (such as building npm packages). Should I create a JIRA for tracking? I think there might be room for reductions.

Reply:
Yes, you should create a JIRA in dev-infra, honestly we need to come up with a process for this type of auditing, or even an automation ideally.

Reply:
Done.